source/documentation/at-rules/import.html.md.erb

```diff
@@ -18,6 +18,48 @@ required to have quotes.
 
 [indented syntax]: ../syntax#the-indented-syntax
 
+<% heads_up do %>
+  Continued use of the `@import` rule is discouraged. It will be [gradually
+  phased out][] over the next few years, and eventually removed from the
+  language entirely. The [`@use` rule][] should be used instead.
+
+  [gradually phased out]: https://github.com/sass/sass/blob/master/accepted/module-system.md#timeline
+  [`@use` rule]: use
+
+  #### What's Wrong With `@import`?
+
+  The `@import` rule has a number of serious issues:
+
+  * `@import` makes all variables, mixins, and functions globally accessible.
+    This makes it very difficult for people (or tools) to tell where anything is
+    defined.
+
+  * Because everything's global, libraries are forced to add prefixes to all
+    their members to avoid naming conflicts.
+
+  * [`@extend` rules][] are also global, which makes it difficult to predict
+    which style rules will be extended.
+  
+    [`@extend` rules]: extend
+
+  * Each stylesheet is executed and its CSS emitted *every time* it's
+    `@import`ed, which increases compilation time and produces bloated output.
+
+  * There was no way to define private members or placeholder selectors that
+    were inaccessible to downstream stylesheets.
+
+  All of these are addressed in `@use` and the new module system.
```

Comment:
``The new module system and the `@use` syntax addresses these problems.``
(active voice)